src/main/java/mate/academy/intro/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package mate.academy.intro.dto;
 
+import jakarta.validation.constraints.Min;
 import lombok.Data;
+import lombok.NonNull;
 
 @Data
 public class CreateBookRequestDto {
+    @NonNull
     private String title;
+    @NonNull
     private String author;
+    @NonNull
     private String isbn;
```

Comment:
With literals better using NotBlank. Do you think you should validate the length?